dotcom-rendering/src/components/FrontSubNav.importable.tsx

```diff
@@ -0,0 +1,69 @@
+import { css } from '@emotion/react';
+import { StraightLines } from '@guardian/source-development-kitchen/react-components';
+import useSWR from 'swr';
+import type { EditionId } from '../lib/edition';
+import { useEditionSwitcherBanner } from '../lib/useUserPreferredEdition';
+import type { SubNavType } from '../model/extract-nav';
+import { Section } from './Section';
+import { SubNav } from './SubNav.importable';
+
+const key = 'edition-switcher-banner';
+const apiPromise = new Promise<{ hidden: boolean }>(() => {
+	/* this never resolves */
+});
+
+type Props = {
+	subNavSections: SubNavType;
+	currentNavLink: string;
+	hasPageSkin: boolean;
+	pageId: string;
+	userEdition: EditionId;
+	currentPillarTitle?: string;
+};
+
+export const FrontSubNav = ({
```

Comment:
Moved into its own component so we can use the `useEditionSwitcherBanner` hook. The `<Subnav />` was already in an island; this does move the straight lines into the island too.